libobs/media-io/video-io.c

```diff
@@ -371,13 +394,37 @@ static inline void reset_frames(video_t *video)
 	os_atomic_set_long(&video->total_frames, 0);
 }
 
+static const video_t *get_const_root(const video_t *video)
+{
+	while (video->parent)
+		video = video->parent;
+	return video;
+}
+
+static video_t *get_root(video_t *video)
+{
+	while (video->parent)
+		video = video->parent;
+	return video;
+}
+
 bool video_output_connect(
 	video_t *video, const struct video_scale_info *conversion,
 	void (*callback)(void *param, struct video_data *frame), void *param)
+{
+	return video_output_connect2(video, conversion, 0, callback, param);
```

Comment:
Did you mean to use a divisor of 1? video_output_connect2() seems programmed to fail when you pass 0 to it

Reply:
You're right, missed updating this when changing from `skip_frames` to `frame_rate_divisor`

Reply:
fixed in https://github.com/obsproject/obs-studio/pull/9001/commits/0c1c17056952549c68fcbe9726584fe636bf1e06